### PR TITLE
feat: Added home page

### PR DESCRIPTION
**Pull request by ScoobyDubeyDo**
Screenshot (138)
Screenshot (139)

### CODE REVIEW

**Review by mohammadkhizerkhan — APPROVED** [review]
Clean code good naming convention,
Good to merge

**Review by znjs — APPROVED** [review]
- Clean Code
- Good folder Structure
- liked that you explored `module css`
- Good Naming Conventions

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                        />
+                    </Link>
+                </div>
+                <div className={`${styles["search-box"]}`}>
+                    <input
+                        type="search"
+                        placeholder="Search"
+                        className={`${styles["treeTv-search"]}`}
+                        onChange={(e) => {
+                            console.log("adsddf");
```
Comment (aniketxparihar) [inline src/components/Navbar/Navbar.jsx:39]: remove consoles if not necessary

**Comment by aniketxparihar** [comment]
- Very Clean Code
- Good Naming Convention
- Good Folder Structure
- Nice UI
Good to Merge

**Pull request merged**